aws/src/main/java/org/apache/iceberg/aws/glue/DefaultLockManager.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aws.glue;
+
+/**
+ * Default implementation of {@link LockManager} that always returns true.
+ * It is mainly used for testing purposes or when no external locking systems are needed.
+ */
+public class DefaultLockManager implements LockManager {
```

Comment:
If this is used for testing, then why not implement it as an in-memory lock manager using synchronized methods? It seems strange to have a default lock manager that doesn't perform locking.